### DISCUSSION TITLE
为啥我这用不了 giscus 啊

### DISCUSSION THREAD

**Discussion by zirnc** [WingLim/hugo-tania#57] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42088872/138557433-eafb9da7-82d4-442c-b5bd-3b9ceb833217.png)
https://github.com/ChungZH/ChungZH.github.io/blob/44761a82e5036fbb5a847900692c0f77e8c6e8ab/config.yaml#L46

![image](https://user-images.githubusercontent.com/42088872/138557442-8ae6c06e-d873-4b23-b6e2-1452f3659d74.png)

哭哭 😭

**Comment by WingLim** [comment, marked as answer]
https://github.com/ChungZH/ChungZH.github.io/blob/44761a82e5036fbb5a847900692c0f77e8c6e8ab/config.yaml#L47

`enable` 改成 `enabled`，之前修改之后忘记同时更新wiki了，sorry。
